cluster/manifests/skipper/configmap-open-policy-agent.yaml

```diff
@@ -0,0 +1,32 @@
+{{ if eq .ConfigItems.skipper_open_policy_agent_enabled "true" }}
```

Comment:
Please add an entry in `deletions.yaml` to remove the config-map in case it was previously enabled but is disabled again.

See example: https://github.com/zalando-incubator/kubernetes-on-aws/blob/e85e0b03498144f81e7d6488e727435764694a25/cluster/manifests/deletions.yaml#L115-L119